manager/integration/tests/test_settings.py

```diff
@@ -200,7 +204,6 @@ def check_tolerations_set(current_toleration_list, set_tolerations):
     return current_tolerations == set_tolerations
 
 
-@pytest.mark.skip(reason="TODO")
 def test_setting_guaranteed_engine_cpu():
```

Comment:
Why don't use the `client` and `core_api` fixture? Otherwise, we cannot guarantee it's cleaned up correctly.

Reply:
Also, in the `reset_setting` fixture, we should reset the setting back to the default value, and wait for the IM to be recreated correctly.

Reply:
Sure, let me add the fixture.